backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -786,14 +786,18 @@ def _create_config_map(
                 "promtail.yaml": yaml.dump(
                     {
                         "server": {
-                            "http_listen_port": promtail_server_port,
+                            "http_listen_port": cfg["promtail"]["serverPort"],
```

Comment:
Not related to this PR, but the function name is a bit confusing. I'd rename it `_create_promtail_configmap` as it's really promtail specific.

Reply:
Yes totally agree with that.  Applied your naming suggestion!